specs/_features/maxeb_increase/capella.py

```diff
@@ -1134,6 +1161,25 @@ def decrease_balance(state: BeaconState, index: ValidatorIndex, delta: Gwei) ->
     state.balances[index] = 0 if delta > state.balances[index] else state.balances[index] - delta
 
 
+def compute_exit_epoch_and_update_churn(state: BeaconState, exit_balance: Gwei) -> Epoch:
+    earliest_exit_epoch = compute_activation_exit_epoch(get_current_epoch(state))
+    per_epoch_churn = get_validator_churn_limit(state)
+    # New epoch for exits.
+    if state.earliest_exit_epoch < earliest_exit_epoch:
+        state.earliest_exit_epoch = earliest_exit_epoch
+        state.exit_balance_to_consume = per_epoch_churn
+
+    # Exit fits in the current earliest epoch.
+    if exit_balance < state.exit_balance_to_consume:
```

Comment:
```suggestion
    if exit_balance <= state.exit_balance_to_consume:
```

See the comment below